### ISSUE TITLE
Style issue for output cell

### ISSUE THREAD

**Issue by randy3k** [jupyter/notebook#110]
If the output cell is hidden, the "..." button is not placed in the middle. Instead, it is aligned to the left. It only happens on my Mac safari, chrome doesn't have this issue.

![screen shot 2015-05-23 at 11 12 03 pm](https://cloud.githubusercontent.com/assets/1690993/7786920/43e99a5e-01a1-11e5-9117-2b9c0309d185.png)

I am on current master branch. Interestingly, ipython 3.0 doesn't have this issue.

**Comment by randy3k** [comment]
There is also a problem in showing isolated svg on Mac Safari.

```
from IPython.display import SVG, display
display(SVG('http://upload.wikimedia.org/wikipedia/commons/5/59/Usa_counties_large.svg'), metadata=dict(isolated=True))
```

![screen shot 2015-05-23 at 11 34 04 pm](https://cloud.githubusercontent.com/assets/1690993/7786954/3a7b7a34-01a4-11e5-872a-a70841f898fd.png)

Only part of the svg is shown.

**Comment by minrk** [comment]
Sorry for letting this slip through. It appears to be fixed in 4.0. Feel free to reopen if you can still reproduce it.
